Controllers/Backend/MolliePayments.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use Mollie\Api\MollieApiClient;
+use MollieShopware\Components\ApplePayDirect\Services\ApplePayDomainFileDownloader;
+use MollieShopware\Components\Config;
+use MollieShopware\Components\MollieApiFactory;
+use MollieShopware\Components\Services\PaymentMethodService;
+use MollieShopware\Components\Services\ShopService;
+use Psr\Log\LoggerInterface;
+use Shopware\Components\Model\ModelManager;
+use Shopware\Components\Plugin\ConfigReader;
+use Shopware\Components\Plugin\PaymentInstaller;
+
+class Shopware_Controllers_Backend_MolliePayments extends Shopware_Controllers_Backend_ExtJs
+{
+    public function updateAction()
+    {
+        try {
+            /** @var PaymentMethodService $paymentMethodService */
+            $paymentMethodService = $this->container->get('mollie_shopware.payment_method_service');
+
+            if ($paymentMethodService !== null) {
+                // Deactivate all Mollie payment methods
+                $paymentMethodService->deactivatePaymentMethods();
```

Comment:
why are you deactivating molie payments in here?
shouldnt this only be done when deactivating the plugin? 
what if this happens accidentally. its not the place where one would expect this to happen

Reply:
During the request Mollie gets asked which payments are available... when the payments aren't available anymore it should be disabled. But since the paymentMethodService does activate them this should only be a 'problem', if there already IS a problem with this payment method

Reply:
good one, totally fine for me
So we have an auto-deactivation feature, gfoit ma :) danke